src/main/java/org/generationcp/middleware/service/impl/study/StudyServiceImpl.java

```diff
@@ -74,6 +96,15 @@
 	@Resource
 	private StudyDataManager studyDataManager;
 
+	@Resource
+	private ExperimentModelGenerator experimentModelGenerator;
+
+	@Resource
+	private VariableDataValidatorFactory variableDataValidatorFactory;
+
+	@Resource
+	private WorkbenchDataManager workbenchDataManager;
```

Comment:
Avoid using workbenchDataManager, you can directly use equivalent methods at DAO level (daoFactory).